### PR TITLE
INC-160 Add links to DPS

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: integration_tests/mockApis/incentivesApi.ts [hunk 1]
```diff
@@ -21,7 +21,8 @@ export default {
     return stubFor({
       request: {
         method: 'GET',
-        urlPattern: '/incentivesApi/incentives-summary/prison/MDI/location/MDI-42',
+        urlPattern:
+          '/incentivesApi/incentives-summary/prison/MDI/location/MDI-42\\?sortBy=DAYS_SINCE_LAST_REVIEW&sortDirection=DESC',
```
Comment (Mjwillis) [inline integration_tests/mockApis/incentivesApi.ts:25]: Just added this for now to indicate sorting

File: server/data/incentivesApi.test.ts [hunk 2]
```diff
@@ -26,7 +26,9 @@ describe('IncentiveApi', () => {
       const locationId = 'ABC-1'
       const apiResponse = getTestIncentivesLocationSummary({ prisonId, locationId })
       incentivesApi
-        .get(`/incentives-summary/prison/${prisonId}/location/${locationId}`)
+        .get(
+          `/incentives-summary/prison/${prisonId}/location/${locationId}?sortBy=DAYS_SINCE_LAST_REVIEW&sortDirection=DESC`
```
Comment (Mjwillis) [inline server/data/incentivesApi.test.ts:30]: Sample sort example, will replace with params

Comment (xoen) [reply server/data/incentivesApi.test.ts:30]: I left the sorting out as the default seems to be sorting by name (both in terms of Incentives API and in terms of requirements apparently) - but adding this here would mean the default ordering would now be wrong.

File: server/views/pages/incentives-table.njk [hunk 3]
```diff
@@ -69,13 +69,13 @@
...
             { html: '<a href="' + dpsHome + '/prisoner/' + prisoner.prisonerNumber + '" class="govuk-link">' + prisoner.lastName + ', ' + prisoner.firstName + ' (' + prisoner.prisonerNumber + ')</a>' },
-            { text: prisoner.daysOnLevel },
-            { text: prisoner.daysSinceLastReview },
-            { text: prisoner.positiveBehaviours },
-            { text: prisoner.incentiveEncouragements },
-            { text: prisoner.negativeBehaviours },
-            { text: prisoner.incentiveWarnings },
-            { text: prisoner.provenAdjudications }
+            { html: '<a href="' + dpsHome + '/prisoner/' + prisoner.prisonerNumber + '/incentive-level-details?agencyId=' + entries.prisonId + '&incentiveLevel=' + level.levelDescription + '" class="govuk-link">' + prisoner.daysOnLevel + '</a>' },
+            { html: '<a href="' + dpsHome + '/prisoner/' + prisoner.prisonerNumber + '/incentive-level-details?agencyId=' + entries.prisonId + '&incentiveLevel=' + level.levelDescription + '" class="govuk-link">' + prisoner.daysSinceLastReview + '</a>' },
```
Comment (xoen) [inline server/views/pages/incentives-table.njk:73]: This link and the one above are the same (AFAICT) - maybe it's worth having it in a variable - or maybe it's overkill.

**Review by xoen — CHANGES_REQUESTED** [review]
This looks good but the API call to Incentives API changed the default order from "by name" (which I think it's what they want by default) to "by days since last review DESC" - I think this should be changed back.

**Review by xoen — APPROVED** [review]

**Pull request merged**